doc/changelog.md

```diff
@@ -92,11 +92,14 @@
   annotations.
   [(#751)](https://github.com/PennyLaneAI/catalyst/pull/751)
 
-* Refactored `vmap` decorator in order to follow a unified pattern that uses a callable 
-  class that implements the decorator's logic. This prevents having to excessively define 
+* Refactored `vmap` decorator in order to follow a unified pattern that uses a callable
+  class that implements the decorator's logic. This prevents having to excessively define
   functions in a nested fashion.
   [(#758)](https://github.com/PennyLaneAI/catalyst/pull/758)
 
+* Catalyst tests now manipulate device capabilities rather than text configurations files.
+  [(#712)](https://github.com/PennyLaneAI/catalyst/pull/712)
```

Comment:
This change is not obversable for the user in the distribution, so doesn't need a changelog entry. Typically we exclude changelog entries for test-only, ci-only, doc-only, and repo-only changes.

Reply:
ok done

Reply:
I don't think the commit changed anything?